crt_portal/cts_forms/forms.py

```diff
@@ -899,6 +899,28 @@ class Filters(ModelForm):
             'class': 'usa-input'
         })
     )
+    create_date_start = DateField(
+        required=False,
+        label="From:",
+        input_formats=('%Y-%m-%d'),
+        widget=DateInput(format='%Y-%m-%d', attrs={
+            'class': 'datepicker usa-input',
+            'name': 'create_date_start',
+            'min': '2020-01-01',
+            'format': '%Y-%m-%d',
```

Comment:
I'd suggest a hint here so that the user knows which format to input. I tried `01-10-2020` and was soundly rejected.

Reply:
The CrtDateWidget, when cleared displays input format. When value is set, presents it in correct format.